_modules/dotnet-test.js

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env node
+
+/**************************************************************************************************
+ * Imports
+ **************************************************************************************************/
+
+const { spawnSync } = require("child_process");
+const dir           = require("./dir");
+const dotnetBuild   = require("./dotnet-build");
+const dotnetPath    = require("./dotnet-path");
+const echo          = require("./echo");
+const shell         = require("shelljs");
+
+/**************************************************************************************************
+ * Constants
+ **************************************************************************************************/
+
+const COVERAGE_FLAGS = "-p:CollectCoverage=true -p:CoverletOutputFormat=opencover";
+
+/**************************************************************************************************
+ * Variables
+ **************************************************************************************************/
+
+let _ciMode       = false;
+let _filter       = [];
+let _skipClean    = false;
+let _withCoverage = false;
+
+/**************************************************************************************************
+ * Functions
+ **************************************************************************************************/
+
+const dotnetTest = {
+    ciMode(ciMode) {
+        if (ciMode != null) {
+            _ciMode = ciMode;
+        }
+
+        return this;
+    },
+    cmd() {
+        return {
+            args: ["test", "--no-build", "--no-restore"],
+            cmd:  "dotnet",
+            toString() {
+                return `${this.cmd} ${this.args.join(" ")}`;
+            },
+        };
+    },
+    description() {
+        return `Runs dotnet test runner on the ${dotnetPath.solutionPath()} solution (via ${this.cmd().toString()})`;
+    },
+    filter(filter) {
+        if (filter != null) {
+            _filter = filter;
+        }
+
+        return this;
+    },
+    /**
+     * Runs dotnet test synchronously for the given test project
+     *
+     * @param {string} project The name of the test project to be run.
+     * @returns {any} An object containing the project name, exit code, and stdout/stderr if 'ciMode' is true.
+     * @example
+     * // An example of the result object to be returned. Code = 0 signifies a successful run,
+     * // while any other code signifies a failure.
+     *  {
+            code: 1,
```

Comment:
Hmm, weird. I'm not seeing any linter warnings about it, and VS Code seems to pick it up just fine: 
![image](https://user-images.githubusercontent.com/11774799/79515938-edc5fc80-8017-11ea-839c-de44d4e9fd18.png)

I could put them in either way, I just thought it might nice to be able to copy/paste the example (although I'm not sure when you'd need to do this)